packages/cozy-procedures/src/locales/en.json

```
@@ -128,7 +129,12 @@
       "conflict_error": "A file with the same name is already uploaded. Please rename your file.",
       "error": "An error occured during upload"
     },
-
+    "unlink": {
+      "title": "Detach from your application?",
+      "subtitle": "You'll detach this file from your application.",
+      "text": "The file will not be deleted from your Document",
```

```suggestion
      "text": "The file will not be deleted from your documents",
```

or maybe "from Drive" and override by "from Documents" in the customer's project